libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL.h

```diff
@@ -121,10 +121,6 @@ class WiFiClientSecure : public WiFiClient {
     static bool probeMaxFragmentLength(const char *hostname, uint16_t port, uint16_t len);
     static bool probeMaxFragmentLength(const String& host, uint16_t port, uint16_t len);
 
-    // AXTLS compatible wrappers
-    // Cannot implement this mode, we need FP before we can connect: bool verify(const char* fingerprint, const char* domain_name)
-    bool verifyCertChain(const char* domain_name) { (void)domain_name; return connected(); } // If we're connected, the cert passed validation during handshake
-
     bool setCACert(const uint8_t* pk, size_t size);
```

Comment:
Can you please deprecate the `setCACert`, `setCertificate`, and `setPrivateKey` methods too?  They're axTLS compatibility wrappers and hackish.